tools/test-bots

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
I don't think you should change this to `python`. The `zulipbots` code is now officially python3 only, so I would recommend running `./tools/provision -p /usr/bin/python3` to build a python3 venv if you aren't already. So, you should reverse this! :)

Reply:
I agree with Eeshan here - the tests are supposed to run on python3, which is why we have this shebang. If you are on Windows and don't have `python3` in your path, just change the shebang for yourself locally.